src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -54944,6 +54946,8 @@ INSERT INTO tab_container_parameter_rates VALUES(5954,'DISEASE_STATE','CKD','Tar
 INSERT INTO tab_container_parameter_rates VALUES(157,'ORGAN','Kidney','eGFR','GFR_Calculation','PARAM_eGFR',0.0,1,NULL,1,NULL);
 INSERT INTO tab_container_parameter_rates VALUES(5015,'GENERAL','PROTEIN','Disease factor','DiseaseStates','One_Rate',0.0,1,NULL,1,NULL);
 INSERT INTO tab_container_parameter_rates VALUES(5941,'GENERAL','TRANSPORTER','Disease factor','DiseaseStates','One_Rate',0.0,1,NULL,1,NULL);
+INSERT INTO tab_container_parameter_rates VALUES(5955,'EVENTGROUP','Urine compartment emptying','Urine compartment emptying enabled','Events','One_Rate',0.0,1,1.0,1,NULL);

```

Comment:
Why do we need this parameter `Urine compartment emptying enabled`?
Because the users want to be able to activate/deactivate the event in a simulation without reconfiguring?

Reply:
since urine is a compartment already. Why do we have the name compartment?

Reply:
Similar to Gallbladder emptying enabled we do not say Gallbladder compartment xx
Going to rename for consistency sake

Reply:
Actually I think the event and the parameter should be called "Bladder emptying" / "Bladder emptying enabled" 
(or "Urinary bladder emptying" / "Urinary bladder emptying enabled").
Maybe somebody with medical background (@sfrechen ?) can comment on this?

Reply:
Makes sense to me..
Urinary bladder sounds more physiological than urine compartment